.github/generate-quest-metadata.main.kts

```diff
@@ -0,0 +1,535 @@
+/**
+ * Note: Run from the project's root directory like this:
+ * kotlinc -script -Xplugin=/path/to/kotlinx-serialization-compiler-plugin.jar .github/generate-quest-metadata.main.kts -- <our args>
+ * --task wiki|json
+ */
+
+@file:DependsOn("org.jetbrains.kotlinx:kotlinx-serialization-core:1.0.1")
+@file:DependsOn("org.jetbrains.kotlinx:kotlinx-serialization-json:1.0.1")
+@file:Suppress("PLUGIN_IS_NOT_ENABLED")
+
+import kotlin.text.*
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import java.io.File
+import java.net.URL
+
+val projectDirectory = File(".")
+val sourceDirectory = projectDirectory.resolve("app/src/main/java/de/westnordost/streetcomplete/")
+val iconsDirectory = projectDirectory.resolve("res/graphics/quest icons/")
+
+val csvFile = projectDirectory.resolve("quest-list.csv")
+val jsonFile = projectDirectory.resolve("taginfo.json")
+
+val noteQuestName = "OsmNoteQuest"
+val noteQuestFile = sourceDirectory.resolve("data/osmnotes/notequests/OsmNoteQuestType.kt")
+
+val wikiRowSpan2 = " rowspan=\"2\" |"
+
+main(args)
+
+@Serializable
+data class Project(val name: String,
+                   val description: String,
+                   val project_url: String,
+                   val doc_url: String,
+                   val icon_url: String,
+                   val contact_name: String)
+
+@Serializable
+data class TaginfoTag(val key: String,
+                      val value: String,
+                      val description: String,
+                      val doc_url: String,
+                      val icon_url: String)
+
+data class TaginfoChange(val key: String,
+                         val value: String,
+                         val change: String)
+
+@Serializable
+data class Base(val data_url: String,
+                val project: Project,
+                val tags: List<TaginfoTag>,
+                val data_format: Int = 1)
+
+fun main(args: Array<String>) {
+    val task = if (args.contains("--task")) args[1 + args.indexOf("--task")] else "wiki"
+    println("Performing task: " + task)
+
+    val questFileContent = sourceDirectory.resolve("quests/QuestModule.kt").readText()
+    val questNameRegex = Regex("(?<=^ {8})[A-Z][a-zA-Z]+(?=\\()", RegexOption.MULTILINE)
+    val questNames = listOf(noteQuestName) + questNameRegex.findAll(questFileContent).map { it.value }
+
+    val questFiles = getFilesRecursively(sourceDirectory.resolve("quests/"))
+
+    when (task) {
+      "wiki" -> generateWikiCsv(questNames, questFiles)
+      "json" -> generateTaginfoJson(questNames, questFiles)
+      else -> { // Note the block
```

Comment:
What this comment is referring to? `{` `}`?

Reply:
Yeah. I think it's possibly because if I didn't put the block in, it didn't print or something?